src/System.Security.Permissions/src/System/Xaml/Permissions/XamlAccessLevel.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Security;
+
+namespace System.Xaml.Permissions
+{
+    /// <SecurityNote>
```

Comment:
Replace `SecurityNote` here and elsewhere with real document-comments.

Reply:
Deleted comment